typescript/src/string/lcfirst.test.ts

```diff
@@ -0,0 +1,17 @@
+import { lcfirst } from "./lcfirst";
+
+const text = "Shizumu you ni tokete yuku you ni.";
+
+describe("valid parameters", () => {
+  it("should return lowercased first characters from capital text", () => {
+    expect(lcfirst(text)).toEqual("shizumu you ni tokete yuku you ni.");
+  });
+
+  it("should return lowercased first characters from all uppercased text", () => {
+    expect(lcfirst(text.toUpperCase())).toEqual("sHIZUMU YOU NI TOKETE YUKU YOU NI.");
+  });
+
+  it("should return lowercased first characters from all lowercased text", () => {
+    expect(lcfirst(text.toLowerCase())).toEqual("shizumu you ni tokete yuku you ni.");
+  });
+});
```

Comment:
tambahin test case buat invalid parameter, misal null, undefined, empty string, dll

Reply:
done in [6cdf8f3](https://github.com/teknologi-umum/pehape/pull/65/commits/6cdf8f30960242c41d2cfd0adcd4937bce29cc1e)